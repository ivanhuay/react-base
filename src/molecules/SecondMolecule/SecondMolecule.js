import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames/bind';
import * as styles from './SecondMolecule.module.scss';

const cx = classNames.bind(styles);

const SecondMolecule = ({ title }) => (
  <div className={cx('container')}>{title}</div>
);

SecondMolecule.defaultProps = {
  title: '',
};

SecondMolecule.propTypes = {
  title: PropTypes.string,
};
export default SecondMolecule;
