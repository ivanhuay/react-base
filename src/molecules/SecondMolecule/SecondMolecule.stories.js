import React from 'react';

import SecondMolecule from './SecondMolecule';

export default {
  component: SecondMolecule,
  title: 'molecules/SecondMolecule',
};

export const Default = function () {
  return <SecondMolecule>default</SecondMolecule>;
};
