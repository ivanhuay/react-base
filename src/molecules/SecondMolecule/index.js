import SecondMolecule from './SecondMolecule';

export default SecondMolecule;
