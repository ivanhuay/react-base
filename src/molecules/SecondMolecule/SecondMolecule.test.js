import React from 'react';
import { render } from '@testing-library/react';
import SecondMolecule from './SecondMolecule';

test('SecondMolecule render', () => {
  const { asFragment } = render(
    <SecondMolecule title="example" />,
  );
  expect(asFragment()).toMatchSnapshot();
});
