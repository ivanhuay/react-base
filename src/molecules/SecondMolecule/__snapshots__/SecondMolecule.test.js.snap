// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondMolecule render 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    example
  </div>
</DocumentFragment>
`;
